### PR TITLE
test added for GET/api/users

### PR DESCRIPTION
**Pull request by JayashreeSamantasinghar**
### What does this pull request aim to achieve? 💯
Added test for GET/api/users
_Please be concise and link any related issue(s):_

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: mocha/test/users.test.js
Comment (jessica-lau) [inline mocha/test/users.test.js:1]: httpStatus constant isn't being used in the test. I suggest either removing this line or changing line 14 down below from '200' to 'httpStatus.OK'

**Review by jessica-lau — DISMISSED** [review]
Test passed, LGTM. Please see my suggestion in the code :)

**Review by galeza — CHANGES_REQUESTED** [review]
Hi @JayashreeSamantasinghar 
Please, could you rebase or merge staging branch to the feature/api-users. PR #1862 was merged. 
Method `validateStatusCodeResponseBody` was replaced by generic methods:
  ```
  validator.validateResponse(response, {
      statusCode: httpStatus.OK,
    });
    validator.validateResponseBody(response, "firstName", "lastName");
```

**Review by jessica-lau — APPROVED** [review]
@JayashreeSamantasinghar LGTM, retested after you merged staging branch 

image

**Review by galeza — APPROVED** [review]
LGTM

**Comment by adorapham** [comment]
LGTM. TY 
image

**Comment by adorapham** [comment]
LGTM. TY @JayashreeSamantasinghar 
image

**Review by adorapham — APPROVED** [review]

**Review by Selbinyyaz — APPROVED** [review]
Looks great to me. Thank you @JayashreeSamantasinghar 

Screen Shot 2021-01-06 at 2 43 59 PM

**Comment by JayashreeSamantasinghar** [comment]
Thanks Everyone !! @galeza ,can you please merge this PR if you have access.

**Pull request merged**
